src/PanelTraits/Create.php

```diff
@@ -22,12 +25,12 @@ public function create($data)
         $data = $this->decodeJsonCastedAttributes($data, 'create');
         $data = $this->compactFakeFields($data, 'create');
 
-        // ommit the n-n relationships when updating the eloquent item
+        // omit the n-n relationships when updating the eloquent item
         $nn_relationships = array_pluck($this->getRelationFieldsWithPivot('create'), 'name');
         $item = $this->model->create(array_except($data, $nn_relationships));
 
         // if there are any relationships available, also sync those
-        $this->syncPivot($item, $data);
+        $this->createRelations($item, $data);
```

Comment:
I'm not sure how one can make a `protected` (or `private`) method `public` without having PHP throw a language level error.